Install failure

A fresh `npm i` is failing for me.

Is this due to the lack of locked-down dependency definitions?

Full logs:

```sh
npm i

> gcx@0.0.3 prepare /Users/timmerman/Documents/github/JustinBeckwith/gcx
> npm run compile


> gcx@0.0.3 compile /Users/timmerman/Documents/github/JustinBeckwith/gcx
> tsc -p .

src/cli.ts:153:22 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("/Users/timmerman/Documents/github/JustinBeckwith/gcx/node_modules/ora/index")' has no compatible call signatures.

153       const spinny = ora('Initializing deployment...').start();
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/index.ts:272:27 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("/Users/timmerman/Documents/github/JustinBeckwith/gcx/node_modules/globby/index")' has no compatible call signatures.

272       const files = await globby(
                              ~~~~~~~
273           '**/**', {ignore: ignorePatterns, cwd: this._options.targetDir});
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/index.ts:274:21 - error TS7006: Parameter 'f' implicitly has an 'any' type.

274       files.forEach(f => {
                        ~

```

--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
     "build/src",
     "src/.gcloudignore"
   ],
+  "repository": "JustinBeckwith/gcx",
   "scripts": {
     "test": "nyc mocha build/test --require source-map-support/register --timeout 60000",
     "lint": "gts check",
@@ -45,13 +46,11 @@
     "@commitlint/cli": "^7.4.0",
     "@commitlint/config-conventional": "^7.3.1",
     "@types/archiver": "^2.1.2",
-    "@types/globby": "^8.0.0",
     "@types/meow": "^5.0.0",
     "@types/mocha": "^5.2.5",
     "@types/nock": "^9.3.0",
     "@types/node": "^10.12.7",
     "@types/node-fetch": "^2.1.3",
-    "@types/ora": "^3.0.0",
     "@types/proxyquire": "^1.3.28",
     "@types/update-notifier": "^2.5.0",
     "@types/uuid": "^3.4.4",

--- a/src/cli.ts
+++ b/src/cli.ts
@@ -2,7 +2,7 @@
 import * as meow from 'meow';
 import {Deployer, DeployerOptions, ProgressEvent} from './';
 import * as updateNotifier from 'update-notifier';
-import * as ora from 'ora';
+import ora from 'ora';
 import * as util from 'util';
 import * as fs from 'fs';
 import * as path from 'path';

--- a/src/index.ts
+++ b/src/index.ts
@@ -2,7 +2,7 @@ import * as archiver from 'archiver';
 import {EventEmitter} from 'events';
 import * as fs from 'fs';
 import {GaxiosResponse} from 'gaxios';
-import * as globby from 'globby';
+import globby from 'globby';
 import {GoogleAuth, GoogleAuthOptions} from 'google-auth-library';
 import {cloudfunctions_v1, google} from 'googleapis';
 import fetch from 'node-fetch';

